build/backend.js

```diff
@@ -37,7 +37,7 @@ gulp.task('backend', ['package-backend'], function(doneFn) {
         '-i',
         // record version info into src/version/version.go
         '-ldflags',
-        conf.recordVersionExpression,
+        conf.recordVersionExpression + ' -w -s',
```

Comment:
This code is only executed during development. Not to build a release binary or docker image. I think we can keep debug information in this case.

Reply:
That's right, we should keep them for development